SafeguardDevOpsService/Extensions/PemExtensions.cs

```diff
@@ -26,9 +26,9 @@ public static string ToPemFormat(this X509Certificate2 This)
         {
             var b64String = Convert.ToBase64String(This.Export(X509ContentType.Cert), Base64FormattingOptions.None);
             var builder = new StringBuilder()
-                .Append("-----BEGIN CERTIFICATE-----\r\n")
-                .Append(AddPemLineBreaks(b64String))
-                .Append("\r\n-----END CERTIFICATE-----");
+                .AppendLine("-----BEGIN CERTIFICATE-----")
```

Comment:
This won't build properly on Linux.  It will use \n instead because that is the environment newline.  I needed to put it in explicitly, because \r\n is what the spec says.

Reply:
Reverted this one